### DISCUSSION TITLE
How do you install this program?

### DISCUSSION THREAD

**Discussion by jseverson** [MerlinCooper/iRacingReplayDirector#107] [General]
I downloaded zip of github repo, but don't see any obvious exe files to install it. Am I missing something obvious here?

**Comment by FearWhat** [comment]
You want to go here, 
https://github.com/MerlinCooper/iRacingReplayDirector/releases/tag/Release_20230316.1 

You will see a zip file 
iRacingReplayDirector_20230316.1.zip 

unzip that file and there is the program to launch.

**Comment by jseverson** [comment]
Are u interested in turning this into a professional paid product? With a
website and cleaner UI etc? I think there is high demand for this if we can
make it a bit more user friendly. Maybe a subscription model?

I’m a bit of a retired entrepreneur myself. Cofounded chess.com long time
ago and now love to race.

**Reply by MerlinCooper** [reply]
The originator did setup this tool as public domain and free under the open source rules. I don‘t see any possibility to convert that into a paid product w/o violating copyright as long as most of the code remains.

For sure a website and an installation program (including proper code signing) would be a benefit to the user community but looking at the statistics that‘s very likely a small group of people and when you carefully read the readme file it‘s not that hard to get it installed by unzipping the files to your local harddrive.
